src/main/java/com/google/cloud/genomics/dataflow/readers/bam/HeaderInfo.java

```diff
@@ -0,0 +1,250 @@
+/*
```

Comment:
This seems generally useful and not specific to dataflow.  Consider pushing it down to util-java in a subsequent PR.